submodules/wizard/wizard.scss

```diff
@@ -272,21 +272,32 @@ $color-highlight: #dbdbde;
 			height: 38px;
 		}
 
-		// Make chosen dropdown to fill container width
-		.chosen-container {
+		.chosen-container.chosen-container-single {
+			// Make chosen dropdown to fill container width
 			width: auto !important;
 			display: block;
 
 			.chosen-single {
 				height: 38px;
 
 				span {
+					font-size: 14px;
 					line-height: 38px;
 				}
 			}
 
 			.chosen-drop {
 				min-width: 100%;
+
+				.chosen-search input[type="text"] {
+					height: auto;
+				}
+			}
+
+			.monster-country-item-flag {
+				margin-right: 0.75rem;
+				width: 18px;
+				max-height: 20px;
```

Comment:
Shouldn't this be in `monster.ui.scss`?

Reply:
I decided to put these styles here because they are specific for the wizard, because the form fields from the wizard design are bigger and somewhat different than the default ones that are used in Monster UI.